contrib/docker/testnet.sh

```diff
@@ -37,9 +37,20 @@ gid="docker" # Assumes user is in docker group
 
 # TODO(unassigned): better sequencing impl
 #Note: sequence limit [2:254]
+# Sequence for base instances (included in reseed)
 seq_start=10  # Not 0 because of port assignments, not 1 because we can't use IP ending in .1 (assigned to gateway)
-seq_end=$((${seq_start} + 19))  # TODO(unassigned): arbitrary end amount
-sequence="seq -f "%03g" ${seq_start} ${seq_end}"
+seq_base_nb=${KOVRI_NB_BASE:-20}
```

Comment:
Can you keep/adjust the TODO regarding arbitrary amount? We should discuss a theoretically-sound value.